conditional/util/auth.py

```diff
@@ -24,3 +26,13 @@ def wrapped_func(*args, **kwargs):
                      "is_eval": is_eval}, *args, **kwargs)
 
     return wrapped_func
+
+
+def get_username(func):
```

Comment:
Just a thought: Almost every time we use this wrapper we immediately grab their LDAP object, why not just do that in the wrapper and return the user object?